charts/atlantis/values.schema.json

```
@@ -313,6 +313,10 @@
       "type": "string",
       "description": "Default Terraform version to be used by atlantis server"
     },
+    "defaultTFDistribution": {
+      "type": "string",
+      "description": "Default Atlantis distribution to be used by atlantis server. Either Opentufu or terraform"
```

The default value is missing in schema, and we can add enum as well.

```suggestion
      "description": "Default Atlantis distribution to be used by atlantis server. Either Opentufu or terraform",
      "default": "terraform",
      "enum": [
        "terraform",
        "Opentufu"
      ]
```